packages/react/src/components/ComposedModal/next/ComposedModal.js

```diff
@@ -240,31 +240,32 @@ const ComposedModal = React.forwardRef(function ComposedModal(
       onClick={handleClick}
       onKeyDown={handleKeyDown}
       className={modalClass}>
-      {/* Non-translatable: Focus-wrap code makes this `<span>` not actually read by screen readers */}
-      <span
-        ref={startSentinel}
-        tabIndex="0"
-        role="link"
-        className={`${prefix}--visually-hidden`}>
-        Focus sentinel
-      </span>
       <div
-        ref={innerModal}
         className={containerClass}
         role="dialog"
         aria-modal="true"
         aria-label={ariaLabel ? ariaLabel : generatedAriaLabel}
         aria-labelledby={ariaLabelledBy}>
-        {childrenWithProps}
+        {/* Non-translatable: Focus-wrap code makes this `<button>` not actually read by screen readers */}
+        <button
+          type="button"
+          ref={startSentinel}
+          tabIndex="0"
```

Comment:
As a button, this element should have a native tabindex, this line is probably not needed.

Reply:
Yup, just removed thanks!